src/Blocks/BlockButton.h

```diff
@@ -71,6 +71,15 @@ class cBlockButtonHandler :
 		BLOCKTYPE & a_BlockType, NIBBLETYPE & a_BlockMeta
 	) override
 	{
+		BLOCKTYPE Block;
```

Comment:
buttons require same style of compatibility checks within GetPlacementBlockTypeMeta as for torches; see [here](https://github.com/csunday95/cuberite/blob/2ccaa5eb9f41e166d18d24736b9242839f48bc12/src/Blocks/BlockTorch.h#L27)